content/guides/authentication/sso.md

```diff
@@ -38,11 +37,12 @@ Box account.
 ## Custom Apps with Server-side Authentication
 
 In any other situation a Custom App is authenticated using server-side
-technology and no actual Managed User is involved. For this reason, SSO is not
-involved in this process. These applications often
+technology and no actual Managed User is involved. These applications often  
+utilize their own user-facing authentication flows.
 
 This means that for [Custom Apps][custom_app] that use [JWT][jwt] or [App
-Token][app_token] authentication no SSO log-in is involved.
+Token][app_token] authentication, SSO with Box is not involved during the  
+authentication flow.
```

Comment:
Again, let's be more specific here. Maybe we can see "is not part of the authentication flow with Box"? Still feels clunky. 

Maybe "...SSO is not used to authenticate any user to Box"?